ethpm_cli/main.py

```diff
@@ -1,10 +1,28 @@
 from ethpm_cli._utils.logger import cli_logger
-from ethpm_cli.constants import ETHPM_CLI_VERSION
+from ethpm_cli.constants import (
+    BLUE_STRING,
+    COLOR_RESET,
+    ETHPM_CLI_VERSION,
+    GREEN_STRING,
+)
 from ethpm_cli.parser import parser
 
+ETHPM_LOGO = """
+       _   _     ____  __  __
+   ___| |_| |__ |  _ \|  \/  |
+  / _ \ __| '_ \| |_) | |\/| |
+ |  __/ |_| | | |  __/| |  | |
+  \___|\__|_| |_|_|   |_|  |_|
+
+"""  # noqa: W605
```

Comment:
I'm guessing that the logo is too much for now.  I'd suggest not doing this unless it's common in other package managers (which I don't recall seeing very often).

Maybe this could only be displayed in the `version` or `help` commands (and myabe `wizard`)?